transports/sftp/src/test/resources/mule-sftp-connection-returned-to-pool-on-error-config.xml

```diff
@@ -15,7 +15,7 @@
     <sftp:connector name="sftpPooledConnector"
                     pollingFrequency="1000"
                     autoDelete="false"
-                    sizeCheckWaitTime="1"
+                    fileAge="1"
```

Comment:
The related test case has a latch that is counted down when the file is processed (before this test case overrode the canProcessFile method in a Test MessageReceiver). Due to these changes, I have to replace the canProcessFile method (that doesn't exist anymore) with the isOldFile method, and the sizeCheckWaitTime with the fileAge attribute.